jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -82,6 +82,37 @@ backend cluster_monitor
     server kibana<%= index %> <%= ip %>:<%= p("haproxy.cluster_monitor.backend_port") %> check inter 1000
     <% end %>
 
+<% if properties.haproxy.elastic.inbound_port %>
```

Comment:
just to double check: is this line not going to raise an error if the property is not set?